lib/net.c

```diff
@@ -54,6 +52,18 @@ uint8_t default_rss_key[GATEKEEPER_RSS_KEY_LEN] = {
 	0x6a, 0x42, 0xb7, 0x3b, 0xbe, 0xac, 0x01, 0xfa,
 };
 
+/* Function to randomize the default RSS key */
+static void
+randomize_rss_key(void)
```

Comment:
`randomize_rss_key()` needs to validate that the RSS key is a reasonable key. If it's not a reasonable key, we need to get another key and test it again.

To test if a key is reasonable, count the number of bits set to 1 in the key and divide this number by the total number of bits. If this percentage is less than 10% or greater than 90%, that's not a reasonable key.